frontend/global/helpers.js

```diff
@@ -54,9 +65,9 @@ function process(data) {
 
 function period_html(period) {
 	return period
-		? (period.url
+		? period.url
 			? `<a href=${period.url}>${period.friendly_name.replaceAll('<', '&gt;').replaceAll('>', '&lt;')}</a>`
-			: period.friendly_name.replaceAll('<', '&gt;').replaceAll('>', '&lt;'))
+			: period.friendly_name.replaceAll('<', '&gt;').replaceAll('>', '&lt;')
```

Comment:
Another minor thing, the symbols you're replacing are switched in both the replacement for the link and the replacement for just the text, so you're replacing "<" with ">" and ">" with "<".
**Ex**: "< test >" becomes "> test <"
(I've confirmed this)

Reply:
Oops! Will fix this now.